packages/client/src/__tests__/dmmfTypes.test.ts

```diff
@@ -23,7 +23,7 @@ model User {
 }
 `
 
-test('dmmf types', async () => {
+test.skip('dmmf types', async () => {
```

Comment:
This test is now pointless when run in jest via swc.

Reply:
I think that adding a comment in the file would be better here